app/views/users/new.html.erb

```diff
@@ -76,7 +76,7 @@
         <em>* Any request for page access higher then Viewer or data access higher then External researcher must be approved by an administrator</em>
 
         <% if Rails.env != "test" %>
-          <%= recaptcha_tags ssl: true %>
+          <%= recaptcha_tags %>
         <% end %>
```

Comment:
Is the ssl key not needed?

Reply:
This is no longer an option see https://github.com/ambethia/recaptcha#recaptcha_tags